### DISCUSSION TITLE
Linux auditbeat questions

### DISCUSSION THREAD

**Discussion by rserio** [Security-Onion-Solutions/securityonion#7990] [Unsupported Versions]
I see the downloads now contain the auditbeat module which is awesome.  I did the so-allow for my server and I setup a tcpdump and see the server coming in, but I'm not seeing any logs coming in, I check the alerts and the elastic dashboard but I'm still new in figuring these out, I"m just trying to prove that this is a viable solution for all server logs so I can extend this to our administrators.  I am using the logstash configuration for the auditbeat module.

**Comment by dougburks** [comment]
Is your Security Onion box configured as eval, standalone, or distributed?

**Reply by rserio** [reply]
Currently it is standalone.

**Reply by dougburks** [reply]
When you ran `so-allow`, which option did you choose?

**Reply by rserio** [reply]
I did the allow for Logstash Beat, as that is teh configuration I used in the auditbeat.

**Reply by dougburks** [reply]
Have you checked the Logstash logs for additional clues?
https://docs.securityonion.net/en/2.3/logstash.html#diagnostic-logging
